beacon/types/exec_header.go

```diff
@@ -28,18 +29,21 @@ import (
 	"github.com/protolambda/ztyp/tree"
 )
 
+// headerObject is an interface that defines the method to get the HashTreeRoot.
 type headerObject interface {
 	HashTreeRoot(hFn tree.HashFn) zrntcommon.Root
 }
 
+// ExecutionHeader holds a reference to an object that implements the headerObject interface.
```

Comment:
This comment is not informative. It says exactly what one can see at a glance, but nothing useful really.